src/Core/src/Platform/Tizen/WindowExtensions.cs

```diff
@@ -110,5 +111,19 @@ static void OnBackButtonPressed(Window window)
 				s_windowCloseRequestHandler[window].Invoke();
 		}
 
+		internal static Devices.DisplayOrientation GetOrientation(this IWindow window)
+		{
+			int displayWidth = PlatformUtils.GetFeatureInfo<int>("screen.width");
+			int displayHeight = PlatformUtils.GetFeatureInfo<int>("screen.height");
+
+			if (displayHeight >= displayWidth)
+			{
+				return Devices.DisplayOrientation.Portrait;
+			}
+			else
+			{
+				return Devices.DisplayOrientation.Landscape;
+			}
+		}
```

Comment:
@PureWeen 

```suggestion
		internal static Devices.DisplayOrientation GetOrientation(this IWindow window) =>
			window.Handler?.MauiContext?.GetPlatformWindow()?.Rotation switch
			{
				0 => Devices.DisplayOrientation.Portrait,
				90 => Devices.DisplayOrientation.Landscape,
				180 => Devices.DisplayOrientation.Portrait,
				270 => Devices.DisplayOrientation.Landscape,
				_ => Devices.DisplayOrientation.Unknown
			};
```

I think we can use `Window.Rotations` instead as above.
https://samsung.github.io/TizenFX/API9/api/ElmSharp.Window.html#ElmSharp_Window_Rotation

Thanks!